src/Tests/Integration/fixtures/banks/discount/09-dashboard/frame-1.html

```
@@ -0,0 +1,4 @@
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)